scriptmodules/supplementary/wifi.sh

```diff
@@ -150,9 +156,6 @@ _EOF_
 function gui_connect_wifi() {
     _set_interface_wifi down 2>/dev/null
     _set_interface_wifi up 2>/dev/null
-    # BEGIN workaround for dhcpcd trigger failure on Raspbian stretch
-    systemctl restart dhcpcd &>/dev/null
-    # END workaround
```

Comment:
I would probably leave this change out or do it in another commit at least.

Reply:
Ok, sure.